job/server/src/main/java/alluxio/master/job/plan/PlanTracker.java

```diff
@@ -296,4 +299,21 @@ private synchronized boolean removeFinished() {
                 || x.getValue().getPlanInfoWire(false).getName().equals(name)))
         .map(Map.Entry::getKey).collect(Collectors.toSet());
   }
+
+  private void checkActiveSetReplicaJobs(JobConfig jobConfig) throws JobDoesNotExistException {
+    if (jobConfig instanceof SetReplicaConfig) {
+      Set<Pair<String, Long>> activeJobs = mCoordinators.values().stream()
+          .filter(x -> x.getPlanInfo().getJobConfig() instanceof SetReplicaConfig)
+          .map(x -> ((SetReplicaConfig) x.getPlanInfo().getJobConfig()))
+          .map(x -> new Pair<>(x.getPath(), x.getBlockId())).collect(Collectors.toSet());
+      SetReplicaConfig config = (SetReplicaConfig) jobConfig;
+      String path = config.getPath();
+      long blockId = config.getBlockId();
+      Pair<String, Long> block = new Pair<>(path, blockId);
+      if (activeJobs.contains(block)) {
+        throw new JobDoesNotExistException(String.format(
```

Comment:
not sure we should create a new kind of exception or just use the existing one to keep it simple.